regl-camera.js

```diff
@@ -53,9 +58,9 @@ function createCamera (regl, props_) {
       prevX = x
       prevY = y
     })
-    mouseWheel(function (dx, dy) {
-      ddistance += dy / window.innerHeight
-    })
+    mouseWheel(source, function (dx, dy) {
+      ddistance += dy / height
```

Comment:
Should it query the height from within the callback? Otherwise the behavior will be stuck at the *initial* height, won't it?